packages/flutter/test/material/control_list_tile_test.dart

```diff
@@ -34,20 +34,232 @@ void main() {
     expect(log, equals(<dynamic>[false, '-', false]));
   });
 
-  testWidgets('RadioListTile control test', (WidgetTester tester) async {
+  testWidgets('RadioListTile should initialize according to groupValue', (WidgetTester tester) async {
+    final List<int> values = <int>[0, 1, 2];
+    int selectedValue;
+    // Input parameters are required for [RadioListTile], but they are
+    // irrelevant when searching with [find.byType].
+    final Type radioListTileType = const RadioListTile<int>(
+      value: 0,
+      groupValue: 0,
+      onChanged: null,
+    ).runtimeType;
+    List<RadioListTile<int>> generatedRadioListTiles;
+
+    await tester.pumpWidget(wrap(
+      child: StatefulBuilder(
+        builder: (BuildContext context, StateSetter setState) {
+          return Scaffold(
+            body: ListView.builder(
+              itemCount: values.length,
+              itemBuilder: (BuildContext context, int index) => RadioListTile<int>(
+                onChanged: (int value) {
+                  setState(() { selectedValue = value; });
+                },
+                value: values[index],
+                groupValue: selectedValue,
+                title: Text(values[index].toString()),
+              ),
+            ),
+          );
+        },
+      ),
+    ));
+
+    generatedRadioListTiles = find
+      .byType(radioListTileType)
+      .evaluate()
+      .map((Element element) {
+        final RadioListTile<int> tile = element.widget;
+        return tile;
+      }).toList();
+
+    expect(generatedRadioListTiles[0].checked, equals(false));
+    expect(generatedRadioListTiles[1].checked, equals(false));
+    expect(generatedRadioListTiles[2].checked, equals(false));
+
+    selectedValue = 1;
+
+    await tester.pumpWidget(wrap(
```

Comment:
Instead of repeating the code to build a frame here you could define a local function:

```
Widget buildFrame() {
  return wrap(
    child: StatefulBuilder(
      builder: (BuildContext context, StateSetter setState) {
        return Scaffold(
          body: ListView.builder(
            itemCount: values.length,
            itemBuilder: (BuildContext context, int index) => RadioListTile<int>(
              onChanged: (int value) {
                setState(() { selectedValue = value; });
              },
              value: values[index],
              groupValue: selectedValue,
              title: Text(values[index].toString()),
            ),
          ),
        );
      },
    ),
  );
}
```